Source: http://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/media-video/avidemux/files/avidemux-2.5.4-build-plugins-fix.patch?revision=1.1&view=markup
Upstream: No
Reason: Fixes plugin build (probably due to the way we build plugins)

--- avidemux_2.5.4.orig/CMakeLists.txt
+++ avidemux_2.5.4/CMakeLists.txt
@@ -205,7 +205,7 @@ ADD_SUBDIRECTORY(avidemux)
 ADD_SUBDIRECTORY(po)
 ADD_SUBDIRECTORY(scripts)
 # MEANX : DOES NOT WORK ADD_SUBDIRECTORY(plugins EXCLUDE_FROM_ALL)
-
+ADD_SUBDIRECTORY(plugins EXCLUDE_FROM_ALL)
 ########################################
 # Config Summary
 ########################################
